flipperdevices/flipperzero-firmware
# [FL-3383, FL-3413] Archive and file browser fixes

# What's new

- Fixed file list reload if list position was changed during last load

# Verification 

- Open any folder with > 1000 files in any app with file browser
- Select the first line in list
- Quickly press Up - Down
- File list should load successfully
- Repeat the same for archive browser

# Checklist (For Reviewer)
